javascript.md

```diff
@@ -55,7 +60,11 @@ let bar = thing; // throws, undefined
 Rust:
 
 ```rust
-// TODO
+if true {
```

Comment:
is it idiomatic in rust to not include `( )` after if and before `{`? 

I wonder, is there like a JSCS equivalent for style in Rust?